src/NServiceBus.Core/Recoverability/FailedConfig.cs

```diff
@@ -9,16 +12,36 @@ public class FailedConfig
         /// Creates a new fault configuration.
         /// </summary>
         /// <param name="errorQueue">The address of the error queue.</param>
+        [ObsoleteEx(ReplacementTypeOrMember = "FailedConfig(string errorQueue, HashSet<Type> unrecoverableExceptionTypes)")]
```

Comment:
Don't we need the treat as error from attribute as well to not make this a breaking change?